src/components/NavBar/index.tsx

```diff
@@ -32,16 +32,18 @@ export default class NeracoosNavBar extends React.Component<object, State> {
 
   public render() {
     return (
-      <div>
+      <div className="neracoos-nav">
         <Navbar dark={true} expand="md">
-          <NavbarBrand tag={Link} to={paths.home}>
-            <img src={neracoosLogo} alt="NERACOOS" />
-          </NavbarBrand>
+          <Link href={"/"} passHref={true}>
+            <NavbarBrand>
+              <img src={neracoosLogo} alt="NERACOOS" />
+            </NavbarBrand>
+          </Link>
           <NavbarToggler onClick={this.toggle} />
 
           <Collapse isOpen={this.state.isOpen} navbar={true}>
             <Nav className="ml-auto" navbar={true}>
-              <NavLink className="nav-link" activeClassName="active" to={paths.home} exact={true}>
+              <NavLink className="nav-link" activeClassName="active" to={paths.home}>
```

Comment:
Codacy found an issue: ['NavLink' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6288046)